### ISSUE TITLE
infinite boot loop after update to macOS 10.15.4 

### ISSUE THREAD

**Issue by glowingkitty** [hellodeibu/gpd-pocket2-hackintosh#14]
A few days ago I installed the most recent 10.15.4 update.... which made it impossible to boot normally again... for a while at least I was able to boot into safe mode. But now (I don't know why) even that doesn't work.... Seems I have a time machine backup on a USB stick from about 2 weeks ago. Not perfect but I guess better than completely reinstalling macOS...
But does anyone know how I can fix the boot loop since which I have now?  Or do I have to use the time machine backup and install that with the help of a real macbook?

**Comment by daveschz** [comment]
What was your version before updating? Was this a point update (from 10.15.3 to 10.15.4)?

**Comment by glowingkitty** [comment]
Updated from 10.15.3 to 10.15.4, yes. With the clover config @THEDEVIOUS1 created, which is based on the config of @hellodeibu  - see https://github.com/hellodeibu/gpd-pocket2-hackintosh/issues/8

**Comment by glowingkitty** [comment]
I reinstalled MacOS Mojave (10.14.6) now (since I wasn't able to make MacOS 10.15.4 work and don't know where I can find the installer / updater for 10.15.3). But now I cannot boot the latest updated clover config @THEDEVIOUS1 created but instead have to use the older version by @hellodeibu - where all the fixes we implemented (sound, 4k display, sleep etc.) don't work. Can anyone help?:(

UPDATE:
I realized by accident that I still have a USB stick with 10.15.3 - and was able to update to that version. Now everything seems to work just before the update to 10.15.4 :) But I hope without the sudden unexpected crashes which I had the last weeks with 10.15.3 and the latest clover config from @THEDEVIOUS1 ...
